definitions/infos.hcl

```diff
@@ -13,20 +13,24 @@ info "object" "meta" "etag" {
 info "object" "meta" "id" {
   type    = "string"
   export  = true
-  comment = "ID is the unique key in service."
+  comment = "ID is the unique key in storage."
 }
 info "object" "meta" "name" {
   type    = "string"
   export  = true
-  comment = "Name is the relative path towards service's WorkDir."
+  comment = "Name is either the absolute path or the relative path towards storage's WorkDir."
```

Comment:
`Name is either the absolute path or the relative path towards storage's WorkDir depends on user's input.`
Maybe add some explaination?